packages/plugin-commands-patching/src/patch.ts

```diff
@@ -1,35 +1,63 @@
+import fs from 'fs'
 import path from 'path'
 import { docsUrl } from '@pnpm/cli-utils'
 import { Config, types as allTypes } from '@pnpm/config'
 import { LogBase } from '@pnpm/logger'
-import { createOrConnectStoreController, CreateStoreControllerOptions } from '@pnpm/store-connection-manager'
+import {
+  createOrConnectStoreController,
+  CreateStoreControllerOptions,
+} from '@pnpm/store-connection-manager'
 import parseWantedDependency from '@pnpm/parse-wanted-dependency'
 import pick from 'ramda/src/pick'
 import pickRegistryForPackage from '@pnpm/pick-registry-for-package'
 import renderHelp from 'render-help'
 import tempy from 'tempy'
 
-export const rcOptionsTypes = cliOptionsTypes
+export function rcOptionsTypes () {
+  return pick([], allTypes)
+}
 
 export function cliOptionsTypes () {
-  return pick([], allTypes)
+  return { ...rcOptionsTypes(), path: String }
+}
+
+export const shorthands = {
+  p: '--path',
 }
 
 export const commandNames = ['patch']
 
 export function help () {
   return renderHelp({
     description: 'Prepare a package for patching',
-    descriptionLists: [],
+    descriptionLists: [{
+      title: 'Options',
+      list: [
+        {
+          description: 'set a package directory path for patching',
```

Comment:
I think this option name is confusing. Maybe something like `--extract-dir` would be better.

The description is also not clear to me. Also it should be capitalized.

Reply:
I get it. How about 'user-dir'?

Reply:
no, maybe `--edit-dir` or `--modify-dir`

Reply:
I changed the name to '--edit-dir'